internal/sidekick/internal/rust_release/publish_test.go

```diff
@@ -239,7 +239,48 @@ func TestPublishPlanMismatchError(t *testing.T) {
 	}
 	remoteDir := setupForPublish(t, "release-2001-02-03")
 	cloneRepository(t, remoteDir)
-	if err := Publish(config, true); err == nil {
+	if err := Publish(config, true, false); err == nil {
 		t.Fatalf("expected an error during plan comparison")
 	}
 }
+
+func TestPublishSkipSemverChecks(t *testing.T) {
+	requireCommand(t, "git")
+	requireCommand(t, "/bin/echo")
+	tmpDir := t.TempDir()
+	// Create a fake cargo that fails on `semver-checks`
+	cargoScript := path.Join(tmpDir, "cargo")
+	script := `#!/bin/bash
+if [ "$1" == "semver-checks" ]; then
+	exit 1
+elif [ "$1" == "workspaces" ] && [ "$2" == "plan" ]; then
+	echo "google-cloud-storage"
+else
+	/bin/echo $@
+fi
+`
+	if err := os.WriteFile(cargoScript, []byte(script), 0755); err != nil {
```

Comment:
@amanda-tarafa my guess is that this won't work well on Windows?

Reply:
I will probably skip this test for now on windows.

I think it might work to just add ".exe" but I'm not sure about if the script will run so I would need to get on my windows machine to test it.